packages/controller/lib/setup/setupSetup.js

```diff
@@ -252,7 +254,7 @@ Please DO NOT copy files manually into ioBroker storage directories!`
         dbConnect(params, async (_objects, _states) => {
             objects = _objects;
             states = _states;
-            const iopkg = fs.readJsonSync(`${__dirname}/../../io-package.json`);
+            const iopkg = fs.readJsonSync(`${CONTROLLER_DIR}/io-package.json`);
```

Comment:
path.join pls

Reply:
I totally agree with avoiding the simple string path. I think it would be the best to harmonize it for the whole repo at some point. The focus of this PR is to only avoid the dependence on the current directory, so I wanted to avoid touching all the changed lines again, because then there will be still other occurrences in the Repo. If I find time + motivation I can optimize it for this PR.

Reply:
got it